src/main.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+playerConfig = sys.argv[1].split("=")[1]
+
+f = open("./config/" + playerConfig, "r")
+config = f.read().splitlines()
+board = []
```

Comment:
Board should not be global I think. It should be declared and returned from `initBoard` function